React/Modules/RCTStatusBarManager.m

```diff
@@ -52,9 +83,12 @@ static BOOL RCTViewControllerBasedStatusBarAppearance()
 
 - (void)startObserving
 {
+#if !TARGET_OS_TV
```

Comment:
Agreed, I'll back out all the TARGET_OS_TV stuff here and just stub this out in the tvOS project files when we get there.